apps/dashboard/src/components/subscribers/subscriber-row.tsx

```diff
@@ -38,33 +38,28 @@ type SubscriberRowProps = {
   subscriber: SubscriberResponseDto;
 };
 
-type SubscriberLinkTableCellProps = ComponentProps<typeof TableCell> & {
-  subscriber: SubscriberResponseDto;
-};
-
-const SubscriberLinkTableCell = (props: SubscriberLinkTableCellProps) => {
-  const { subscriber, children, className, ...rest } = props;
-  const { currentEnvironment } = useEnvironment();
+type SubscriberLinkTableCellProps = ComponentProps<typeof TableCell>;
 
-  const editSubscriberLink = buildRoute(ROUTES.EDIT_SUBSCRIBER, {
-    environmentSlug: currentEnvironment?.slug ?? '',
-    subscriberId: subscriber.subscriberId,
-  });
+const SubscriberTableCell = (props: SubscriberLinkTableCellProps) => {
+  const { children, className, ...rest } = props;
 
   return (
     <TableCell className={cn('group-hover:bg-neutral-alpha-50 text-text-sub relative', className)} {...rest}>
       {children}
-      <Link to={editSubscriberLink} className={cn('absolute inset-0')}>
```

Comment:
This was the root problem for the truncated text popover not opening. Basically, the transparent link element was positioned absolutely occupying the whole cell. 
Now the click is happening on the whole row, and we do prevent the other "internal" clicks from propagating to the row, I'll mention them later in the comments.